src/components/Donations/Donations.js

```diff
@@ -83,9 +85,7 @@ const Donations = () => {
         )}
 
         {status === 'success' && !(data.pages[0].data.length > 0) && (
-          <p className="my-11 w-full flex justify-center items-center">
-            No se han encontrado donaciones para mostrar
-          </p>
+        <EmptyState icon={<EmptyStateDonation height="10.813rem"/>} text="No se han encontrado donaciones" />
```

Comment:
already handled on the EmptyState component

```suggestion
        <EmptyState icon={<EmptyStateDonation />} text="No se han encontrado donaciones" />
```